tests/params/test_tensor.py

```diff
@@ -0,0 +1,243 @@
+import numpy as np
+import pytest
+
+import porepy as pp
+
+
+def fourth_order_tensor_for_testing(
+    num_cells: np.array, return_tensor_and_matrix: bool
+):
+    """Construct a fourth order tensor with custom fields for testing.
+
+    The tensor which is constructed here is for an anisotropic and homogeneous medium.
+
+    Parameters:
+        return_tensor_and_matrix: Flag telling whether only the tensor should be
```

Comment:
We can also always return both arguments, and dump the second one if not needed. This is arguably simpler code, but feel free to fix as you see fit.